modules/apps/frontend-js/frontend-js-remote/tests/RemoteAppliactions.testcase

```diff
@@ -0,0 +1,82 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-js-components-sample-web";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Remote Applications";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		PortalInstances.tearDownCP();
+	}
+
+	@description = "LPS-163527. Assert iFrame is not add and warnings appear."
+	@priority = "4"
+	test AssertErrorMessegeAppearsWhenIFrameFieldsEmpty {
```

Comment:
HasMandatoryFieldErrorMessagesWhenEmpty

see for more tips: https://liferay.slack.com/archives/CLD1FFVL6/p1618527187013400?thread_ts=1618516694.010400&cid=CLD1FFVL6